internal/cloudapi/openapi.yml

```diff
@@ -110,18 +110,45 @@ components:
       required:
         - status
         - type
+        - target_results
       properties:
         status:
           type: string
           enum: ['success', 'failure', 'pending', 'running']
         type:
           $ref: '#/components/schemas/UploadTypes'
+        target_results:
+          type: array
+          items:
+            $ref: '#/components/schemas/TargetResult'
     AWSUploadStatus:
       type: object
       properties:
         ami_id:
           type: string
           example: 'ami-0c830793775595d4b'
+    TargetResult:
```

Comment:
I woudn't expose this. I think the `TargetResults` from the Job should be converted into an `UploadStatus`.